main.py

```diff
@@ -0,0 +1,123 @@
+import discord, datetime, time, random # Imports discord.py, datetime, time and random libraries
+from discord.ext import commands # Import the extension for discord.py - discord.ext command
+from discord import Member # import discord member, used in warning
+import json # import the json library
+import os

```

Comment:
Would you mind explaining to me what the need to import "os" is and where it functions, thank you

Reply:
its for setting up dirs. the bot functions with a dir called .botmem. os can check if it exists, mkdir and check if warnings.json exists too

Reply:
Okay thank you for your explaination!